docs.css

```diff
@@ -56,6 +62,10 @@ i {
   width: 70%;
 }
 
+.information > *:where(:not(:first-child)) {
+  margin-top: 1rem;
```

Comment:
While using a complex selector it's preferable to add a comment explaining the logic.